### PR TITLE
Feat: Added Home & Authentication

### PR DESCRIPTION
**Pull request by ramalingaa**
Added Authentication and Home page
 - User can now login/signup
 - Landing page is enabled

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]
Code Looks great!
I can see you've done a lot of work!
code can be merged!

File: src/frontend/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  const toggleProfileCard = () => {
+    setProfileDisplay((prev) => !prev)
+  }
+  return (
+    <nav className="navbar">
+    <Link to = "/" className="navbar-logo">
+         <h2>Habitude</h2>
+    </Link>
+    
+     {/* <div className ="navbar-search">
```
Comment (ravikumar1002) [inline src/frontend/components/navbar/Navbar.jsx:18]: remove this comment

**Review by ravikumar1002 — APPROVED** [review]
looks good Ramalinga,
nice folder structure, 
and one thing ram,  try to split your app.css file according to components because it's too big

**Review by PeeyushPrashant — APPROVED** [review]
clean code
good folder structure

File: src/frontend/components/navbar/Navbar.jsx (hunk 1 above)
Comment (ramalingaa) [reply src/frontend/components/navbar/Navbar.jsx:18]: Made Changes

**Pull request merged**